dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -111,7 +111,7 @@ ANNO_CLASSSOURCE_NOT_FILE.useraction=No user action is available.
 # # {4} the name of the class
 ANNO_CLASSSOURCE_OPEN3_EXCEPTION=CWWKC0010W: [ {0} ]: The open of [{1}] failed, for resource [{2}] under root [{3}] for class [{4}].
 ANNO_CLASSSOURCE_OPEN3_EXCEPTION.explanation=The attempt to open an InputStream for the file caused an exception.
-ANNO_CLASSSOURCE_OPEN3_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_OPEN3_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.